docs/docs/graphql.md

```diff
@@ -17,7 +17,7 @@ There's three types of GraphQL operations, but here we'll only focus on two: Que
 The following GraphQL query:
 
 ```graphql
-query GetProject {
+query GetProjectl
```

Comment:
Was this one a mistake? The original seems ok

Reply:
Thanks, I tried to do these edits via the web interface. DO NOT RECOMMEND

Reply:
Fixed